components/salsa-macro-rules/src/setup_tracked_fn.rs

```diff
@@ -184,8 +193,7 @@ macro_rules! setup_tracked_fn {
                     ) -> &$zalsa::interned::IngredientImpl<$Configuration> {
                         let zalsa = db.zalsa();
                         $INTERN_CACHE.get_or_create(zalsa, || {
-                            <dyn $Db as $Db>::zalsa_register_downcaster(db);
```

Comment:
I'm not exactly sure why this was here before, I don't think it's necessary. It's clearer now because we register the downcaster on the function ingredient directly.